README.md

```diff
@@ -68,7 +68,7 @@ _Finally_ we can run `npm start` and start building out our application!
   - attach a screenshot (or a gif!) for design reference if needed
   - reference the related issue
   	- "closes #123" or "fixes #123" will close issue #123 once the PR is merged
-  	- "issue #123" just refences the issue. Only use this if you definitly need the issue to remain open.
+  	- "issue #123" just references the issue. Only use this if you definitively need the issue to remain open.
```

Comment:
`definitly` is actually meant to be an emphasis [`definitely`](https://www.vocabulary.com/articles/chooseyourwords/definitely-definitively/)